src/hammer-vlsi/synthesis/vivado/__init__.py

```diff
@@ -0,0 +1,39 @@
+from hammer_vlsi import HammerSynthesisTool, HammerToolStep, deepdict
+
+from typing import Dict, List, Any
+
+import os
+
+class VivadoSynth(HammerSynthesisTool):
```

Comment:
We should probably add a `defaults.yml` to allow hammer to locate the vivado binary

Reply:
Generally speaking, is hammer supposed to work without sourcing tool-specific scripts? I.e., should I have to source vivado's settings64.sh, or by having the correct defaults.yml should hammer just work?

Reply:
It's desirable since it turns to be possible to waste a lot of time in digging through `source.sh`-esque scripts and it's nice to encode exactly what in the world we need to run a tool (encodes information & reduces messiness). (Of course you could always pre-source stuff and run hammer so we don't remove any functionality, but it's not ideal for re-use.)

Reply:
OK, I'm not sure how to do this but it seems reasonable